src/seq/nucleotide.jl

```diff
@@ -123,6 +123,16 @@ function ispyrimidine(nt::Nucleotide)
 end
 
 """
+    isppambiguous(nt::Nucleotide)
+
+Test if nucleotide could be either a Purine or a Pyrimidine.
+"""
+function isppambiguous(nt::Nucleotide)
```

Comment:
Is this really worth defining & exporting from Bio.Seq? There is no abstraction here.